app/scripts/components/academy/academy-dashboard.components.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {Link} from 'react-router';
+import TutorialContent from 'tutorial-content';
+
+export default class AcademyDashboard extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {};
+		this.tutorials = new TutorialContent();
+	}
+
+	render() {
+		const titles = {
+			home: "Academy",
+			course: "academy",
+		};
+
+		this.tutorials.content.map((tutorial) => {
+			titles[tutorial.slug] = tutorial.title;
+		});
+
+		const backlinks = {
+			home: "/dashboard",
+			course: "/academy",
+		};
+		const curRoute = this.props.location.pathname.split('/')[this.props.location.pathname.split('/') . length - 1];
```

Comment:
I guess I could replace this by
`curRoute = this.props.routeParams.courseSlug || this.props.route.name`